test/TestProjects/MgmtDiscriminator/Generated/Models/DeliveryRuleListResult.Serialization.cs

```diff
@@ -37,8 +95,55 @@ internal static DeliveryRuleListResult DeserializeDeliveryRuleListResult(JsonEle
                     value = array;
                     continue;
                 }
+                if (options.Format != "W")
+                {
+                    additionalPropertiesDictionary.Add(property.Name, BinaryData.FromString(property.Value.GetRawText()));
+                }
+            }
+            serializedAdditionalRawData = additionalPropertiesDictionary;
+            return new DeliveryRuleListResult(Optional.ToList(value), serializedAdditionalRawData);
+        }
+
+        private BinaryData SerializeBicep(ModelReaderWriterOptions options)
+        {
+            StringBuilder sb = new StringBuilder();
+            sb.AppendChildObject(Value, options);
```

Comment:
The namespace that AppendChildObject lives in will need to be added to the usings for bicep serializable models.